packages/two_dimensional_scrollables/test/table_view/table_test.dart

```diff
@@ -1730,6 +1730,478 @@ void main() {
+      final Map<TableVicinity, ({int start, int span})> bothMerged =
+          <TableVicinity, ({int start, int span})>{
+        TableVicinity.zero: (start: 0, span: 2),
+        const TableVicinity(row: 1, column: 0): (start: 0, span: 2),
+        const TableVicinity(row: 0, column: 1): (start: 0, span: 2),
+        const TableVicinity(row: 1, column: 1): (start: 0, span: 2),
+      };
+
+      final Map<TableVicinity, ({int start, int span})> rowMerged =
+          <TableVicinity, ({int start, int span})>{
+        const TableVicinity(row: 2, column: 0): (start: 2, span: 2),
+        const TableVicinity(row: 3, column: 0): (start: 2, span: 2),
+        const TableVicinity(row: 4, column: 1): (start: 4, span: 3),
+        const TableVicinity(row: 5, column: 1): (start: 4, span: 3),
+        const TableVicinity(row: 6, column: 1): (start: 4, span: 3),
+      };
+
+      final Map<TableVicinity, ({int start, int span})> columnMerged =
+          <TableVicinity, ({int start, int span})>{
+        const TableVicinity(row: 0, column: 2): (start: 2, span: 2),
+        const TableVicinity(row: 0, column: 3): (start: 2, span: 2),
+        const TableVicinity(row: 1, column: 4): (start: 4, span: 3),
+        const TableVicinity(row: 1, column: 5): (start: 4, span: 3),
+        const TableVicinity(row: 1, column: 6): (start: 4, span: 3),
+      };
+      const TableSpan span = TableSpan(extent: FixedTableSpanExtent(75));
+
+      testWidgets('Normal axes', (WidgetTester tester) async {
+        final ScrollController verticalController = ScrollController();
+        final ScrollController horizontalController = ScrollController();
+        final TableView tableView = TableView.builder(
+            verticalDetails: ScrollableDetails.vertical(
+              controller: verticalController,
+            ),
+            horizontalDetails: ScrollableDetails.horizontal(
+              controller: horizontalController,
+            ),
+            columnCount: 20,
+            rowCount: 20,
+            pinnedRowCount: 2,
+            pinnedColumnCount: 2,
+            columnBuilder: (_) => span,
+            rowBuilder: (_) => span,
+            cellBuilder: (_, TableVicinity vicinity) {
+              return TableViewCell(
+                columnMergeStart: bothMerged[vicinity]?.start ??
+                    columnMerged[vicinity]?.start,
+                columnMergeSpan:
+                    bothMerged[vicinity]?.span ?? columnMerged[vicinity]?.span,
+                rowMergeStart:
+                    bothMerged[vicinity]?.start ?? rowMerged[vicinity]?.start,
+                rowMergeSpan:
+                    bothMerged[vicinity]?.span ?? rowMerged[vicinity]?.span,
+                child: Text(
+                  'R${bothMerged[vicinity]?.start ?? rowMerged[vicinity]?.start ?? vicinity.row}:'
+                  'C${bothMerged[vicinity]?.start ?? columnMerged[vicinity]?.start ?? vicinity.column}',
+                ),
+              );
+            });
```

Comment:
nitty-nit: trailing comma here for slightly nicer formatting with less indentation?

```suggestion
            },);
```

Reply:
same elsewhere in this file.